### DISCUSSION TITLE
UE Bug

### DISCUSSION THREAD

**Discussion by LukeKl** [community/community#19639] [Projects and Issues] [answered]
The screen will return to the top when you put a repository into a list.

**Comment by melvincwng** [comment, marked as answer]
@jinjunshi, 您好，对于错误，请向 [GitHub Bug Report](https://support.github.com/contact/bug-report) 报告。

希望这可以帮助你。 

如果此答案有帮助，请将其标记为答案并在 GitHub 上关注我。 谢谢。

**Reply by LukeKl** [reply]
THX

**Reply by melvincwng** [reply]
没问题 ;)

**Reply by melvincwng** [reply]
@jinjunshi 如果没有其他问题，您可以“标记为答案”以结束此讨论。 谢谢你。
